src/setOfPeople.js

```diff
@@ -38,7 +38,15 @@
  * @return {Set}
  */
 function setOfPeople(people) {
-  // write code here
+  const set = new Set();
```

Comment:
create more specific name for the variable

Reply:
The name of the variable is changed:
![image](https://user-images.githubusercontent.com/32244670/63585528-01fc4b80-c5a8-11e9-99f6-02fe90deaf03.png)